inscriptions/ISic001226.xml

```diff
@@ -191,58 +196,30 @@
-                    <supplied reason="lost">— —</supplied>ς Ὑπερ
-                    <supplied reason="lost">β</supplied>
-                    <supplied reason="omitted">ό</supplied>λου
-                    <lb n="14"/>
-                    <supplied reason="omitted">Αἴσ</supplied>χρων Ἀριστοξένου<lb n="15"/>
-                    <supplied reason="lost">Ἀ</supplied>φροδίται.
-                    <lb n="16"/>
-                </ab>
-            </div>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI331409">
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#GC">
                 <ab>
-                    <lb n="1"/>
-
-
-                    ναύ<supplied reason="omitted">κλη</supplied>ροι
-                    <lb n="2"/>
+                    <lb n="1"/>ναύ<supplied reason="omitted">κλη</supplied>ροι
+                    <lb n="2"/><name>Ὄλυμπις</name> <name>Ὑπερβόλου</name>
+                    <lb n="3"/><name><unclear>Θ</unclear>εῦγνις</name> <g ref="#interpunct">·</g> <name>Εὐβίου</name>
+                    <lb n="4"/><name>Φρυνείδας</name> <name>Τεισάνδρου</name>
+                    <lb n="5"/><name><supplied reason="omitted">Δι</supplied>οκ<supplied reason="omitted">λῆ</supplied>ς</name> <name>Υσ<supplied reason="lost">—</supplied>του</name>
+                    <lb n="6"/><g ref="#interpunct">·</g> <name>Ἀριστόδαμος</name> <g ref="#interpunct">·</g> <name>Εὐβίου</name>
+                    <lb n="7"/><name>Ογηαουκος</name> <!-- I didn't find any attestation for this name (?) --> <g ref="#interpunct">·</g> <name>Εὐφε<supplied reason="lost">ίδ</supplied>εος</name>
+                    <lb n="8"/><supplied reason="lost">—</supplied>ΞΡΙα<supplied reason="omitted">ρ</supplied>χος <name>Πε<supplied reason="omitted">ι</supplied><unclear>θ</unclear><surplus>ι</surplus>ά<unclear>ρ</unclear>χου</name> <surplus>Πειθάρχου</surplus>
+                    <lb n="9"/><name>Πεί<supplied reason="omitted">σ</supplied>ανδρος</name> <name>Ἀγά<unclear>θ</unclear>ωνος</name>
+                    <lb n="10"/><supplied reason="lost">—</supplied>κ<supplied reason="omitted">λ</supplied>είδας <name>Τεισάνδρου</name>
+                    <lb n="11"/><name><supplied reason="omitted">Ἀπ</supplied>ο<supplied reason="omitted">λ</supplied>λόδωρος</name> <name>Ἀρχεδάμου</name>
+                    <lb n="12"/><name><supplied reason="omitted">Ζώ</supplied>πυρος</name> <name>Ναυκράτεος</name>
+                    <lb n="13"/><supplied reason="lost">— —</supplied>ς <name>Ὑπερ<supplied reason="lost">β</supplied><supplied reason="omitted">ό</supplied>λου</name>
+                    <lb n="14"/><name><supplied reason="omitted">Αἴσ</supplied>χρων</name> <name>Ἀριστοξένου</name>
+                    <lb n="15"/><supplied reason="lost">Ἀ</supplied>φροδίται.
                 </ab>
             </div>
             <div type="apparatus">
                 <p><!--external apparatus criticus (if applicable)--></p>
```

Comment:
You need to spell out which text you're following. Easiest is probably to go for Kaibel's interpreted, and then report Gualtherus' original text as an app note in capitals for each line, e.g.:
`<app loc="line 5"><note>Gualtherus: ΟΚΑΙΣΥΣ . . ΤΟΥ</note></app>`